src/gt4py/backend/numpy_backend.py

```diff
@@ -334,10 +341,17 @@ def numpy_is_compatible_type(field):
 
 @gt_backend.register
 class NumPyBackend(gt_backend.BaseBackend, gt_backend.PurePythonBackendCLIMixin):
-    """Pure Python backend using numpy for faster computations than the debug backend."""
+    """Pure Python backend using numpy for faster computations than the debug backend.
+
+    Options
+    -------
+        ignore_np_errstate: `bool`
+            If False, does not ignore numpy floating-point errors.
+            (`True` by default.)
```

Comment:
Not sure if this is the right place, but I figure it's good to document it somewhere.